### PR TITLE
More SUSE compliance

### PR DESCRIPTION
**Pull request by pseiler**
* building on SUSE with python3 is now working
* Python3 subpackages are just prefixed with python3. No minor version
  suffix
* some python2 packages still named as "python" without 2 as suffix f.e.
  python-devel
* python3 bytecode is generated automatically on SUSE
* wrong python-zypp packages names got fixed
* drop support for SLES11 SP4
* building with python3 is now standard (python2 already does not work
  anymore in a test setup)

### CODE REVIEW

**Comment by pseiler** [comment]
I had pretty much work to do last week I'll update it the next days

File: subscription-manager.spec [hunk 1]
```diff
@@ -74,7 +74,11 @@
 %global python_sitearch %python3_sitearch
 %global python_sitelib %python3_sitelib
 %global __python %__python3
+%if 0%{?suse_version}
```
[the remark this answers is not in the document]
Comment (ptoscano) [reply subscription-manager.spec:77]: @sbernhard now that #2854 is in, this version check is not needed (SUSE >= 15 is assumed)

**Comment by jirihnidek** [comment]
@pseiler I'm considering closing this PR, because #2877 was opened and it doesn't have conflicts as this PR has. Do you agree?

**Comment by pseiler** [comment]
@jirihnidek Absolutely, Sorry for not commenting til now. I thought the other PR already was merged.
Thank you for all the effort. I will test the new stuff and if anything is missing/etc. I'll open another issue and pr.

**Pull request closed without merging**